editor/src/uuiui/styles/theme/utopia-theme.ts

```diff
@@ -111,7 +136,7 @@ const noticeStyles: { [styleName: string]: React.CSSProperties } = {
   },
   info: {
     backgroundColor: '#f1f1f1',
-    color: colorTheme.darkPrimary.value,
+    color: colorTheme.darkPrimary.value as string,
```

Comment:
Is this `as string` necessary here? I believe this is something you said was necessary at some point during this work but has since become redundant.

Reply:
Yes no longer needed